### PR TITLE
React

### PR DESCRIPTION
**Pull request by nathfreder**
## What does this Pull Request do?
This Pull Request ports the repository to React 16.0.0. This will make it easier to host on (GitHub Pages, Surge, Now, Heroku). You will also experience mobile support.

## Will it cause any problems with the site?
No, it will make the site faster and better for production

## Can I make edits?
Sure, Drew. Go ahead.

### CODE REVIEW

**Comment by nathfreder** [comment]
@DrewD3V

File: src/views/NotFound.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './NotFound.css';
+
+class NotFound extends Component {
+  render() {
+    return (
+    <div>
+        <h1>Error code: 404, page not found.</h1>
+        <h2>Let's be honest, did you expect something better for this page?</h2>
```
Comment (nathfreder) [inline src/views/NotFound.js]: This is no problem, GitHub does not know JSX.

File: src/views/Home.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          <Link to="https://discordapp.com/invite/n8mFWED"><button className="discord">Discord</button></Link>
+        </div>
+        </header>
+        <div className="bio">
+            <a href="https://twitter.com/AppleDevelops">
+                <img className="picture" alt="Beauty Drew" src="https://pbs.twimg.com/profile_images/921137292818964486/KBAbftwS_400x400.jpg"/>
+            </a>
+            { /* TODO: Add a bio and more to make this look better I guess. */ }
+            <p>I'm a 15 year old developer.</p>
+         </div>
```
Comment (nathfreder) [inline src/views/Home.js]: This is no problem, GitHub does not know JSX.

**Comment by DrewD3V** [comment]
Looks good, just need to test it later to see compatibility.

**Comment by nathfreder** [comment]
@DrewD3V Alright.

**Comment by nathfreder** [comment]
Alright. Opening a new Pull Request

**Comment by nathfreder** [comment]
Where the fuck did time go..

**Comment by DrewD3V** [comment]
it apparently took me 8 days to consider it and 2 hours to work on it 🤷‍♂️

**Pull request closed without merging**